src/status_im/ui/screens/wallet/send/views.cljs

```diff
@@ -221,10 +221,17 @@
         [components/amount-selector {:disabled?     modal?
                                      :error         (or amount-error
                                                         (when-not sufficient-funds? (i18n/label :t/wallet-insufficient-funds)))
-                                     :input-options {:default-value  (str (money/to-fixed (money/wei->ether amount)))
-                                                     :max-length     21
+                                     :input-options {:max-length     21
                                                      :on-focus       (fn [] (when (and scroll @scroll) (utils/set-timeout #(.scrollToEnd @scroll) 100)))
-                                                     :on-change-text (update-amount-fn timeout)}}]
+                                                     :on-change-text (update-amount-fn timeout)
```

Comment:
Would it make sense to refactor the whole `input-options` so that there is no duplication?

Reply:
it does, there is quite a bit of duplication there, both with views and this logic.
can we do that in a follow-up PR just to unblock this release?

Reply:
there is already a TODO in the request part to refactor it ;-)

Reply:
Ah you mean my TODO ? ;) I guess I was hoping you would fix it :D

Reply:
Yeah, I'm planning to follow-up on it tomorrow.
I think it is a good exercise in Clojure for me :)